gnovm/stdlibs/time/timezoneinfo.go

```diff
@@ -0,0 +1,10 @@
+package time
+
+import (
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+)
+
+func X_chainTZParamData(m *gno.Machine) (data []byte) {
+	_ = m
+	return []byte{}
+}
```

Comment:
```suggestion
func X_chainTZParamData() []byte {
	return nil
}
```

Reply:
@moul Ideally we'd have this load from the params keeper, but it seems tricky to do this and probably not ideal (because the `*SDKParams` which is passed down to the VM only has setters).

Do you think it's OK to only have the stub, and assume different chains would have a different implementation here?